slurmci-test.toml

```diff
@@ -0,0 +1,40 @@
+# Tests to be run by SLURM CI
+#
+# Each entry specifies the number of MPI ranks to use followed by the file to run
+
+cpu = [
+    ["3", "examples/DGmethods_old/ex_001_periodic_advection.jl"],
+    ["3", "examples/DGmethods_old/ex_002_solid_body_rotation.jl"],
+    ["1", "test/Ocean/shallow_water/GyreDriver.jl"],
+]
+
+gpu = [
+    ["3", "examples/DGmethods_old/ex_001_periodic_advection.jl"],
+    ["3", "examples/DGmethods_old/ex_002_solid_body_rotation.jl"],
+    ["3", "examples/DGmethods_old/ex_003_acoustic_wave.jl"],
+    ["3", "examples/DGmethods_old/ex_004_nonnegative.jl"],
+    ["3", "examples/Microphysics/ex_1_saturation_adjustment.jl"],
+    ["3", "examples/Microphysics/ex_2_Kessler.jl"],
+    ["3", "test/DGmethods/advection_diffusion/pseudo1D_advection_diffusion.jl"],
```

Comment:
Please pass the "true" command line argument through to this test

Reply:
That test exists in both `cpu_gpu` with no parameters, and in `gpu` with parameters. This means it will run 3 times: on the CPU with no parameters, on the GPU with no parameters, and on the GPU with the "true" parameter. Is that what is wanted?

Reply:
I've assumed the answer is no, so this test is now in `cpu` with no parameters and in `gpu` with the `true` parameter.

Reply:
Ah, I see you've fixed this in https://github.com/climate-machine/CLIMA/pull/468 as well as the other point discussed below so I'll mark this resolved.